mms/tests/integration_tests/onnx_mxnet_export_integ_test.py

```diff
@@ -0,0 +1,39 @@
+import json
```

Comment:
this whole test seems redundant. what does it do that previous test doesn't?

Reply:
@jesterhazy It tests individual models from onnx-mxnet model zoo. None of the previous test do so